lib/archive/stripper_test.go

```diff
@@ -132,17 +132,19 @@ func TestSingleStripper(t *testing.T) {
 		}
 
 		// make the buf just that little bit bigger to allow for errrors in the copy if they would occur
-		buf := make([]byte, size+10)
+		buf := make([]byte, size)
```

Comment:
you need to remove the comment above if you've dropped the extra spacing. Why did you drop the extra spacing?